pkgs/development/ocaml-modules/reason-native/default.nix

```diff
@@ -0,0 +1,183 @@
+{ lib, fetchFromGitHub, buildDunePackage, ocamlPackages }:
+
+let
+  shared = {
+    version = "a33f152";
```

Comment:
Please pick a version string such that 1. you can easily extract it from the name (i.e. starts with a digit) and 2. will increase (in the `versionAtLeast` order) when you update the packages. Suggestion: `2020-07-18-a33f152`.

Reply:
Thanks. Done.